MantidPlot/src/Mantid/AlgorithmDockWidget.cpp

```diff
@@ -79,17 +84,31 @@ void AlgorithmDockWidget::updateProgress(void *alg, const double p,
   }
 }
 
+void AlgorithmDockWidget::updateDetailsButton() const {
+  std::ostringstream mess;
+  if (m_algID.size() > 0) {
+    mess << "Running " << m_algID.size();
```

Comment:
You could consider just using `+` and for the `size_t` conversion [`std::to_string`](http://de.cppreference.com/w/cpp/string/basic_string/to_string) .